runtime/src/test/resources/tailcall/runtime/graphql/test-modify-interface-field.graphql

```diff
@@ -0,0 +1,38 @@
+#> server-sdl
+schema {
+  query: Query
+}
+
+interface IA {
+  a: String
+}
+
+type B implements IA {
+  a: String @modify(name: "x")
+  b: String
+}
+
+type Query {
+  bar: B
+}
+
+#> client-sdl
+schema {
+  query: Query
+}
+
+interface IA {
+  a: String
+}
+
+type B implements IA {
+  b: String
+  x: String
+}
+
+type Query {
+  bar: B
+}
+
+#> client-error
```

Comment:
Not sure if we should call them as client-errors. I think those should be issued only at run time, when we issue a real graphql query. Something else, perhaps - "validations"?